langchain/schema/language_model.py

```diff
@@ -48,6 +52,75 @@ class BaseLanguageModel(Serializable, ABC):
     Each of these has an equivalent asynchronous method.
     """
 
+    def invoke(
```

Comment:
BaseLanguageModel is actually implementing all methods to take advantage of the native async, batch and stream capabilities of the underlying APIs